src/v/v8_engine/api.cc

```diff
@@ -52,4 +55,48 @@ bool api::is_enabled() {
     return local_cfg.enable_v8() && local_cfg.developer_mode();
 }
 
+ss::future<std::error_code>
+api::insert(model::topic_namespace topic, data_policy dp) {
+    if (!is_enabled()) {
```

Comment:
I think these types of checks should happen at the highest level, like for instance, where a fetch is made and the decision to use v8 or not is checked. It should just occur once in one place, in my opinion

Reply:
It is problem, because insert and remove are used from data_policy_manager, which handle events from raft log.

Reply:
Sorry, I think if user write broken config (on one node enable_v8 = true, on another off) I can not handle this situation. I think it user's problem